src/index.js

```diff
@@ -1334,6 +1365,9 @@ function valueCalculator(
     case 'gallonsToAcreFeet':
       returnValue = gallonsToAcreFeet(value, precision);
       break;
+    case 'litersToUserPreference':
+      returnValue = litersToUserPreference(value, userPreference, unitType, precision);
```

Comment:
`unitType` and `userPreference` aren't passed to the `valueCalculator` function

Reply:
@cnetten can you resolve this?